elasticdl/python/data/reader/data_reader.py

```diff
@@ -1,11 +1,51 @@
 from abc import ABC, abstractmethod
 
+from elasticdl.python.common.dtypes import MAXCOMPUTE_DTYPE_TO_TF
+
 
 class Metadata(object):
+    """ Metadata of a dataset containing column name and dtype
+
+    Attributes:
+        column_names: A list with column names
+        column_dtypes: A dict where the key is a column name
```

Comment:
the value is dtype of MaxCompute or tf?

Reply:
A dtype of MaxCompute or pandas for CSV file, but not a TF dtype. I will add it into the docstring.